docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to provision a cluster, deploy Kyma, and <!--- update is not among the alpha commands ---> with alpha commands.
```

Comment:
I assume you wanted to write a comment here with `<!---` because something is still missing. But since it's inside the text instead of on a new line, it does show up on the [preview](https://github.com/kyma-project/cli/blob/2bb60b8fcf672962715c0e113132b0ea451e4a44/docs/03-03-examples-alpha.md), so you would need to delete it in the end (or write it on a separate line).

```suggestion
The following examples show how to provision a cluster, deploy Kyma, and <!--- update is not among the alpha commands ---> with alpha commands.
```